static/app/views/settings/organizationGeneralSettings/organizationSettingsForm.spec.jsx

```diff
@@ -161,4 +168,26 @@ describe('OrganizationSettingsForm', function () {
       })
     );
   });
+
+  it('enable PR bot is disabled without GitHub integration', function () {
```

Comment:
nit: I think this could be written like `it disables the "enable PR bot" button without GitHub integration` so it fits with Jests grammatical style